gen/_template/handlers.tmpl

```diff
@@ -89,8 +89,14 @@ func (s *{{ if $op.WebhookInfo }}Webhook{{ end }}Server) handle{{ $op.Name }}Req
 					Security: {{ quote $securityName }},
 					Err: err,
 				}
+				{{- if and $.Config.Error (not $op.WebhookInfo) }}
+				if encodeErrorResponse(s.h.NewError(ctx, err), w, span); err != nil {
```

Comment:
I guess it should be

```suggestion
				if encodeErr := encodeErrorResponse(s.h.NewError(ctx, err), w, span); encodeErr != nil {
```

but I am not sure.

Reply:
Good 👀 👍